pkg/rpcservice/webhook_endpoint_create.go

```diff
@@ -0,0 +1,36 @@
+package rpcservice
+
+import (
+	"context"
+
+	"github.com/stripe/stripe-cli/pkg/requests"
+	"github.com/stripe/stripe-cli/pkg/stripe"
+	"github.com/stripe/stripe-cli/rpc"
+)
+
+// WebhookEndpointCreate create a new webhook endpoint
+func (srv *RPCService) WebhookEndpointCreate(ctx context.Context, req *rpc.WebhookEndpointCreateRequest) (*rpc.WebhookEndpointCreateResponse, error) {
+	userConfig := srv.cfg.UserCfg
+	livemode := false
+
+	key, err := userConfig.Profile.GetAPIKey(livemode)
+	if err != nil {
+		return nil, err
+	}
+
+	err = requests.WebhookEndpointCreate(
+		ctx,
+		stripe.DefaultAPIBaseURL,
+		stripe.APIVersion,
```

Comment:
Seems `stripe.APIVersion` is a constant 2019-03-14. I wonder if it makes sense to use empty string instead so that the webhook endpoint uses the merchant default version? That's how the Dashboard works today, so it could be less surprising I think.